### ISSUE TITLE
Name parts with number less 10 with a leading zero

### ISSUE THREAD

**Issue by olegstepura** [bagder/http2-explained#97]
Now github orders files the wrong way. Isuggest solving this by adding a leading zero to numbers < 10

Before:
----
```
part1.md
part10.md
part11.md
part12.md
part13.md
part14.md
part2.md
part3.md
part4.md
part5.md
part6.md
part7.md
part8.md
part9.md
```

After
----
```
part01.md
part02.md
part03.md
part04.md
part05.md
part06.md
part07.md
part08.md
part09.md
part10.md
part11.md
part12.md
part13.md
part14.md
```

**Comment by bagder** [comment]
Right, but does it really matter in which order github shows them?

**Comment by olegstepura** [comment]
I was reading it on github. Ordering did matter. After part 9 I had to scroll back to top to find part 10. I could think there is no part 10 at all. If there would be a link #98, it would be even better.

**Comment by bagder** [comment]
Oh, you were _reading_ it on github... Can I ask you why you read it here? The book renders much better on the actual book site.

**Comment by olegstepura** [comment]
I found it here and read it here =)

**Comment by bagder** [comment]
Try reading the book at http://http2-explained.haxx.se/ instead =)

**Comment by olegstepura** [comment]
BTW github renders thing very well, so reading it on github is ok. Maybe I'm the only ine who did it this way, but I only suggest after reading *all* book and having some experience. Others may be just axy enough to suggest such things ;)

**Comment by bagder** [comment]
Oh sure, I appreciate the feedback, I don't mean to imply anything else. It's just that a kind of "next chapter" link makes less sense in a PDF version or in the html version as it is rendered on http2-explained.haxx.se so I'm not sure how to do it to make it nice all over.

**Comment by olegstepura** [comment]
Well, it's up to you, actually. I cannot insist. Tha't why I opened an issue, not a pull request: to discuss this option.

**Comment by olegstepura** [comment]
Ok,just close the #98 issue as won't fix ;)
